hack/jenkins/linux_integration_tests_kvm.sh

```diff
@@ -36,5 +36,10 @@ EXTRA_TEST_ARGS="-gvisor"
 mkdir -p cron && gsutil -qm rsync "gs://minikube-builds/${MINIKUBE_LOCATION}/cron" cron || echo "FAILED TO GET CRON FILES"
 sudo install cron/cleanup_and_reboot_Linux.sh /etc/cron.hourly/cleanup_and_reboot || echo "FAILED TO INSTALL CLEANUP"
 
+## download and install gopogh
```

Comment:
Is it possible to make this part of `common.sh`?

Reply:
need to install it differently for windows/mac ...each one gets a different binary.

Reply:
You can use `go env GOARCH` and `go env GOHOSTOS` to generate the correct path.

`uname` works too, but you'll need to lowercase the output.